package.json

```diff
@@ -17,14 +17,14 @@
   "main": "lib/index.js",
   "license": "MIT",
   "dependencies": {
-    "async": "2.1.4",
-    "body-parser": "1.16.0",
+    "async": "^2.6.4",
+    "body-parser": "^1.20.2",
     "colors": "*",
     "cors": "2.8.1",
```

Comment:
Ik zou al deze dependencies dan een een meer permissive (`^`) lock geven. Anders lijkt het net alsof er bewust een onderscheid gemaakt is (die andere packages kunnen gewoon geupdated worden, terwijl de rest specifiek deze versie moet blijven). Dan heb je de volgende keer (waarschijnlijk) ook geen --force meer nodig.